packages/core/src/components/blind/blind.tsx

```diff
@@ -129,12 +125,9 @@ export class Blind {
             'blind-header': true,
             closed: this.collapsed,
           }}
-          type="button"
-          role="button"
-          tabindex="0"
           aria-labelledby={`ix-blind-header-title-${this.id}`}
           aria-controls={`ix-blind-content-section-${this.id}`}
-          aria-expanded={this.collapsed}
+          aria-expanded={`${!this.collapsed}`}
```

Comment:
use binary value please i changed it wrong in the last commit `{!this.collapsed}`

Reply:
fixed